src/ServiceBouncer/MainForm.cs

```diff
@@ -334,9 +334,8 @@ private async Task Connect()
                 toolStripConnectButton.Tag = @"Connected";
                 toolStripConnectButton.Image = Properties.Resources.Disconnect;
 
-                machineHostname = machineHostname.Trim();
-                var isCurrentMachine = machineHostname == "." || Environment.MachineName.Equals(machineHostname, StringComparison.CurrentCultureIgnoreCase);
-                backgroundRefreshSeconds = isCurrentMachine ? 1 : 30;
+                var isLocalMachine = EnvHelper.IsLocalMachine(machineHostname);
```

Comment:
Could we remove the isLocalMachine variable here?

Reply:
done